piece_of_cake.py

```diff
@@ -0,0 +1,18 @@
+def get_recipe_price(prices, optionals=[], **products):
```

Comment:
I would use documantation instead, comments make it look messy
https://swimm.io/blog/code-comments-vs-documentation/